### PR TITLE
E_CLASSROOM-255 [FE/BE] Implement adding a new quiz

### PR DESCRIPTION
**Pull request by ErickDoroy**
### Issue Link
- https://framgiaph.backlog.com/view/E_CLASSROOM-255

### Definition of Done
- [x] Able to add quiz in the admin side

### Related PR:
- https://github.com/framgia/sph-classroom-els-fe/pull/109
#### Main note
for add quiz use postman
- http://localhost:82/api/v1/admin_add
title:
instruction:
category_id:
#### Pages Affected
- N/A

#### Scenarios/ Test Cases
- N/A

### CODE REVIEW

**Comment by CaneteJoash** [comment]
### Working 

- [x] Able to add quiz in the admin side.
- [x] Able to add the new quiz in the database. 

image

-----------
Total testing hr. 1.5 hr.

File: app/Http/Controllers/API/v1/Quiz/QuizController.php [hunk 1]
```diff
@@ -72,4 +73,19 @@ public function adminQuiz(Request $request)
 
         return $this->paginate($admin_quizzes);
     }
+
+    public function adminAdd(StoreQuizRequest $request)
```
Comment (karlolee50) [inline app/Http/Controllers/API/v1/Quiz/QuizController.php]: change the function name because adminAdd is too generic

File: routes/api.php [hunk 2]
```diff
@@ -63,6 +63,7 @@
         Route::get('/followedLog', [StudentActivitiesController::class, 'followedActivities']);
         Route::get('/categories/{category_id}/relatedQuizzes/{quiz_id}', [QuizController::class, 'relatedQuizzes']);
         Route::get('/admin_quizzes', [QuizController::class, 'adminQuiz']);
+        Route::post('/admin_add', [QuizController::class, 'adminAdd']);
```
Comment (karlolee50) [inline routes/api.php]: change the route as well. since admin_add is too generic.

**Review by karlolee50 — CHANGES_REQUESTED** [review]
Review time: 10 minutes
Comment/s: 2

**Comment by catampongan** [comment]
## Approved PR #50 
- [x] Able to add quiz in the admin side.
- [x] Able to add the new quiz in the database. 

--------------
image

-----------
- Total Test Time: 1.5 hr
- Total Iterations: 0
- Total Comments: 0

**Review by karlolee50 — APPROVED** [review]

**Pull request merged**